scripts/deployHLP.ts

```diff
@@ -1,20 +1,29 @@
+import { LedgerSigner } from '@anders-t/ethers-ledger'
+const IS_LEDGER_SIGNER = true
+const LEDGER_ACCOUNT_PATH = "44'/60'/0'/0/0"
+
 export async function deploy(network_name, hre) {
+  const ERC20 = await hre.ethers.getContractFactory('MockToken')
```

Comment:
Note: this can be any contract not necessarily "MockToken" (MockToken.sol is not required)

Reply:
Thanks, will revise